api/tests/integration/tests/reaction/pathway.py

```diff
@@ -18,7 +18,15 @@ def find_diff(a, b):
 input_path = joinPathPy("reactions/", __file__)
 ref_path = joinPathPy("ref/", __file__)
 
```

Comment:
Could you please add "indigo.setOption("json-saving-pretty", True)" to make ket files readable?

Reply:
Thanks for your suggestion. Unfortunately, the files used for this feature are huge, thus beautifying them becomes of a quite questionable benefit. And you don't very often need to see it beautified and online services are at hand in 10 sec.

Reply:
It will help when these files changed by other PR